examples/binary/README.md

```diff
@@ -12,6 +15,21 @@ repo:
     path: /var/artipie/data
 ```
 
-You can send HTTP PUT/GET requests
-to `http://localhost:8080/repo/<filename>` to upload/download a binary file,
-e.g. `http://localhost:8080/repo/libsqlite3.so`.
+After creating the configuration file below, artipie is ready to server as a storage for binary files.
+
+In order to upload a binary file to artipie, send a PUT HTTP request with file contents:
+
+```bash
+echo "hello world" > text.txt
+curl --silent -X PUT --data-binary "@text.txt" http://localhost:8080/repo/text.txt
+```
+
+In order to download a file, send a GET HTTP request:
+
+```bash
+curl -X GET http://localhost:8080/repo/text.txt
+```
+
+#### Advanced option
+
+The binary type of repositories does not have any other opinions available.
```

Comment:
@Sammers21 I'd suggest to change "binary type of repositories" -> "binary repository type"